### ISSUE TITLE
Setup CI build

### ISSUE THREAD

**Issue by szpak** [MatthewTamlin/Retrial#13]
For you own convenience it could be useful to setup CI build (e.g. Travis) for the project (to ensure that the code in master compiles not only on your local machine :), not to mention different Java versions).

And thanks for the project :).

**Comment by MatthewTamlin** [comment]
Good idea, I've been meaning to investigate Bitrise of Drone CI for a while now. I'll try and find some time soon.
